gp-includes/things/translation.php

```diff
@@ -296,9 +296,12 @@ public function for_translation( $project, $translation_set, $page, $filters = a
 
 		$join_type = 'INNER';
 
-		$sort_bys = array('original' => 'o.singular %s', 'translation' => 't.translation_0 %s', 'priority' => 'o.priority %s, o.date_added DESC',
-			'random' => 'o.priority DESC, RAND()', 'translation_date_added' => 't.date_added %s', 'original_date_added' => 'o.date_added %s',
-			'references' => 'o.references' );
+		$sort_bys = array();
+		$sort_by_fields = gp_get_sort_by_fields();
+
+		foreach ( $sort_by_fields as $key => $field ) {
+			$sort_bys[ $key ] = $field['sql_sort_by'];
+		}
```

Comment:
This can be changed to `$sort_bys = wp_list_pluck( gp_get_sort_by_fields(), 'sql_sort_by' )`.